### PR TITLE
Add balance to the process

### PR DESCRIPTION
**Pull request by antho1404**
dependency: https://github.com/mesg-foundation/js-sdk/pull/186, https://github.com/mesg-foundation/engine/pull/1723

- [x] add transfer message to the account api
- [x] add cli command `account:transfer`
- [x] create a transfer to the process when created

### CODE REVIEW

File: packages/cli/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,8 @@
 - [#183](https://github.com/mesg-foundation/js-sdk/pull/183) Add account commands `account:balance`, `account:create`, `account:export`, `account:list`
 - [#181](https://github.com/mesg-foundation/js-sdk/pull/181) Add util to get hash from a transaction log
 - [#186](https://github.com/mesg-foundation/js-sdk/pull/186) Create/delete process based on LCD endpoint
+- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Add `account:transfer` command
+- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Transfer token when create a process
```
Comment (NicolasMahe) [inline packages/cli/CHANGELOG.md]:
```suggestion
- [#187](https://github.com/mesg-foundation/js-sdk/pull/187) Transfer token when creating a process
```

File: packages/cli/src/commands/account/transfer.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import { IAccount } from '@mesg/api/lib/account-lcd'
+import { ICoin } from '@mesg/api/lib/transaction'
+
+export default class AccountTransfer extends Command {
+  static description = 'Transfer token to another address'
+
+  static flags = {
+    ...Command.flags,
+    account: flags.string({
+      description: 'Account to use to send the token'
```
Comment (NicolasMahe) [inline packages/cli/src/commands/account/transfer.ts]:
```suggestion
      description: 'Account to use to take the tokens from'
```

File: packages/cli/src/commands/account/transfer.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  }, {
+    name: 'AMOUNT',
+    required: true
+  }]
+
+  async run(): Promise<IAccount> {
+    const { args, flags } = this.parse(AccountTransfer)
+    const { account, mnemonic } = await this.getAccount(flags.account)
+
+    this.spinner.start(`Sending ${args.AMOUNT}atto to ${args.TO}`)
```
Comment (NicolasMahe) [inline packages/cli/src/commands/account/transfer.ts]:
```suggestion
    this.spinner.start(`Transfering ${args.AMOUNT}atto to ${args.TO}`)
```

File: packages/cli/src/commands/account/transfer.ts [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      amount: args.AMOUNT,
+      denom: 'atto'
+    }]
+    const tx = await this.lcd.createTransaction(
+      [this.lcd.account.transferMsg(account.address, args.TO, coins)],
+      account
+    )
+    const txResult = await this.lcd.broadcast(tx.signWithMnemonic(mnemonic))
+    const res = await this.lcd.account.get(account.address)
+    this.styledJSON(res)
```
Comment (NicolasMahe) [inline packages/cli/src/commands/account/transfer.ts]: why displaying the balance of the from account?
let's just display an success message like:
```
`Transfer of ${args.AMOUNT}atto from ${account.address}  to ${args.TO} executed with success`
```

File: packages/cli/src/commands/process/create.ts [hunk 5]
```diff
@@ -36,6 +37,23 @@ export default class ProcessCreate extends Command {
...
     this.spinner.stop(hash)
-    return this.lcd.process.get(hash)
+    const process = await this.lcd.process.get(hash)
+    this.createTransfer(process, mnemonic)
+    return process
+  }
+
+  async createTransfer(process: IProcess, mnemonic: string) {
+    const acc = await this.lcd.account.import(mnemonic)
+    this.spinner.start('Transfer credits to the process')
```
Comment (NicolasMahe) [inline packages/cli/src/commands/process/create.ts]: why need to use only one word for coins/tokens/credits. The rest of the commands are currently using "tokens"
```suggestion
    this.spinner.start('Transfering tokens to the process')
```

File: packages/cli/src/commands/process/create.ts [hunk 6]
```diff
@@ -36,6 +37,23 @@ export default class ProcessCreate extends Command {
     if (hashes.length != 1) throw new Error('error while getting the hash of the process created')
     const hash = hashes[0]
     this.spinner.stop(hash)
-    return this.lcd.process.get(hash)
+    const process = await this.lcd.process.get(hash)
+    this.createTransfer(process, mnemonic)
+    return process
+  }
+
+  async createTransfer(process: IProcess, mnemonic: string) {
```
Comment (NicolasMahe) [inline packages/cli/src/commands/process/create.ts]: is this still necessary? it's done by the process keeper already: https://github.com/mesg-foundation/engine/pull/1699/files#diff-7e9b501b837ce29ef68a80f1cfc21534

Comment (antho1404) [reply packages/cli/src/commands/process/create.ts]: removed

File: packages/cli/src/commands/account/transfer.ts (hunk 4 above)
Comment (antho1404) [reply packages/cli/src/commands/account/transfer.ts]: simply stopped the spinner with a message `success`. This anyway will disappear on the next PR

File: packages/cli/src/commands/process/create.ts [hunk 7]
```diff
@@ -36,6 +37,7 @@ export default class ProcessCreate extends Command {
     if (hashes.length != 1) throw new Error('error while getting the hash of the process created')
     const hash = hashes[0]
     this.spinner.stop(hash)
-    return this.lcd.process.get(hash)
-  }
+    const process = await this.lcd.process.get(hash)
```
Comment (NicolasMahe) [inline packages/cli/src/commands/process/create.ts:40]: this change doesn't need to be but not blocking

**Review by NicolasMahe — APPROVED** [review]

**Pull request merged**